Mods/src/PuppiMod.cc

```diff
@@ -38,53 +38,62 @@ PuppiMod::PuppiMod(const char *name, const char *title) :
   fApplyLowPUCorr(kTRUE),
   fUseEtaForAlgo(kTRUE),
   fEtaForAlgo(2.5),
-  fDumpingPuppi(kFALSE)
+  fDumpingPuppi(kFALSE),
+  fMapper(0)
 {
   // Constructor.
 }
 
 //--------------------------------------------------------------------------------------------------
-PuppiMod::~PuppiMod() {} 
+PuppiMod::~PuppiMod()
+{
+}
 
 //--------------------------------------------------------------------------------------------------
-Int_t PuppiMod::GetParticleType(const PFCandidate *cand)
+PuppiMod::ParticleType
```

Comment:
Here the return type is not an integer but the enum declared in the class definition.